airflow/contrib/operators/sagemaker_endpoint_config_operator.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.operators.sagemaker_base_operator import SageMakerBaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.exceptions import AirflowException
+
+
+class SageMakerEndpointConfigOperator(SageMakerBaseOperator):
+
+    """
+    Create a SageMaker endpoint config.
+    
+    This operator returns The ARN of the endpoint config created in Amazon SageMaker
+
+    :param config: The configuration necessary to create an endpoint config.
+    
+        For details of the configuration parameter, See:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.create_endpoint_config
+    :type config: dict
+    :param aws_conn_id: The AWS connection ID to use.
+    :type aws_conn_id: str
+    """  # noqa
```

Comment:
Why the `noqa?`

Reply:
Just the external link is too long. I am not sure if there's a way to separate a long link to multiple lines.

Reply:
Ah, in that case we can ignore that one specific error

```suggestion
    """  # noqa: E501
```

(I guessed at the error code, so check I got the right one)

Reply:
Got it! Thanks a lot!

Reply:
Updated